### DISCUSSION TITLE
小米HyperOS下启动服务失败

### DISCUSSION THREAD

**Discussion by cool2008** [2dust/v2rayNG#3524] [General]
在提出问题前请先自行排除服务器端问题，同时也请通过搜索确认是否有人提出过相同问题。

### 预期行为
启动服务中会成功

### 实际行为
小米HyperOS下启动服务失败

### 复现方法
1.点击启动
2.
3.

### 日志信息

<details>
通过`adb logcat -s com.v2ray.ang GoLog V2rayConfigUtilGoLog Main`获取日志。请自行删减日志中可能出现的敏感信息。

如果问题可重现，建议先执行`adb logcat -c`清空系统日志再执行上述命令，再操作重现问题。
```
在这里粘贴日志
```
C:\Users\Administrator\Desktop\platform-tools-latest-windows\platform-tools>adb logcat -s com.v2ray.ang GoLog V2rayConfigUtilGoLog Main
--------- beginning of main
08-21 21:58:19.902 20737 20737 I com.v2ray.ang: Using CollectorTypeCC GC.
08-21 21:58:19.907 20737 20737 E com.v2ray.ang: Not starting debugger since process cannot load the jdwp agent.
08-21 21:58:20.259 20737 20772 D com.v2ray.ang: MiuiProcessManagerServiceStub setSchedFifo
08-21 21:58:30.521 20850 20850 I GoLog   : loading core config
08-21 21:58:30.521 20850 20884 I GoLog   : Preparing Domain: free.2apzhfa.xyz:31245
08-21 21:58:30.523 20850 20850 I GoLog   : infra/conf/serial: failed to parse json config > infra/conf: unknown cipher method: ss
08-21 21:58:30.523 20850 20850 D com.v2ray.ang: go.Universe$proxyerror: infra/conf/serial: failed to parse json config > infra/conf: unknown cipher method: ss
08-21 21:58:30.626 20850 20884 I GoLog   : Prepare Result:
08-21 21:58:30.626 20850 20884 I GoLog   :  Domain: free.2apzhfa.xyz
08-21 21:58:30.626 20850 20884 I GoLog   :  Port: 31245
08-21 21:58:30.626 20850 20884 I GoLog   :  IPs: [120.232.217.88]
08-21 21:58:57.654 20737 20747 I com.v2ray.ang: This is non sticky GC, maxfree is 33554432 minfree is 8388608
08-21 21:58:57.655 20737 20749 W com.v2ray.ang: ApkAssets: Deleting an ApkAssets object '<empty> and /product/app/MiuiContentCatcherMIUI15/MiuiContentCatcherMIUI15.apk' with 1 weak references
08-21 21:58:57.655 20737 20749 W com.v2ray.ang: ApkAssets: Deleting an ApkAssets object '<empty> and /system_ext/priv-app/RtMiCloudSDK/RtMiCloudSDK.apk' with 1 weak references
08-21 21:58:57.655 20737 20749 W com.v2ray.ang: ApkAssets: Deleting an ApkAssets object '<empty> and /system_ext/app/miuisystem/miuisystem.apk' with 1 weak references
</details>

### 环境信息
Device: Xiaomi 13
OS: Android 14
OS version: Hyper OS 1.0.10.0.UNCCNXM
V2rayNG version: 1.8.35

**Comment by 13271473920** [comment]
一样的问题  能解决么

**Comment by csc-EricWu** [comment]
能解决吗？

**Comment by csc-EricWu** [comment]
安装后点启动直接闪退，重启系统后，点启动提示成功，但是无法上外网
